metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/AbstractMetisPlugin.java

```diff
@@ -57,6 +58,7 @@
   private String id;
 
   private PluginStatus pluginStatus = PluginStatus.INQUEUE;
+  private String failMessage;
```

Comment:
The getter of this field is missing, which is required for jaxb.

Reply:
Good point. I fixed it.